test/webRDFS/WebServerTest.cc

```diff
@@ -45,6 +45,44 @@ TEST(WebServerTest, testRead) {
   system("hdfs dfs -fs hdfs://localhost:5351 -rm /fileToRead");
 }
 
+TEST(WebServerTest, testCreate) {
+  system(
+        "hdfs dfs -fs hdfs://localhost:5351 "
+        "-copyFromLocal fileForTesting /fileToCreate");
+
+      ASSERT_EQ(0,
+                system("curl -i --insecure https://localhost:8080/webhdfs/v1/"
```

Comment:
`op=CREATE` should not be respected by HTTP GET, according to [spec](https://hadoop.apache.org/docs/r1.0.4/webhdfs.html#CREATE). This should be `curl -X PUT` and the corresponding logic for invoking create should be in `put_handler`

Reply:
fixed